_config.yml

```diff
@@ -6,7 +6,7 @@
 # 'jekyll serve'. If you change this file, please restart the server process.
 
 # Site settings
-title: SECRETSALES.com Tech Blog
+title: Tech Blog
```

Comment:
I prefer having secretsales in the title - just "Tech Blog" on the tab seems weird